var/spack/repos/builtin/packages/py-atlannot/package.py

```diff
@@ -10,29 +10,36 @@
 class PyAtlannot(PythonPackage):
     """Alignment of brain atlas annotations."""
 
-    homepage = 'https://bbpgitlab.epfl.ch/project/proj101/atlas_annotation'
-    git = 'git@bbpgitlab.epfl.ch:project/proj101/atlas_annotation.git'
-
-    maintainers = ['EmilieDel', 'Stannislav']
-
-    version('0.1.1', commit='5271681de15f23e1df6f8556405b03218541d241')
-
-    depends_on('python@3.7:', type=('build', 'run'))
-    depends_on('py-setuptools', type='build')
-    depends_on('py-setuptools-scm', type='build')
+    homepage = "atlas-annotation.rtfd.io"
+    maintainers = ["EmilieDel", "Stannislav"]
+
+    version(
+        "0.1.2",
+        url="https://files.pythonhosted.org/packages/08/b9/6edff732ad711e600b91d32052604987b744f054c049739df73c07ef0232/atlannot-0.1.2.tar.gz",
+        sha256="145dac874752d5e5d093b977aa04798916daf31d1e942cc64d059175deea27df",
+    )
+    version(
+        "0.1.1",
+        tag="v0.1.1",
+        git="git@bbpgitlab.epfl.ch:project/proj101/atlas_annotation.git",
+    )
+
+    # Setup requirements
+    depends_on("python@3.7:", type=("build", "run"))
+    depends_on("py-setuptools", type="build")
+    depends_on("py-setuptools-scm", type="build")
 
     # Installation requirements
-    depends_on('py-antspyx@0.2.4', type=('run'))
-    depends_on('py-atldld@0.2.2', type=('run'))
-    # depends_on('py-dvc', type=('run'))
-    depends_on('py-matplotlib', type=('run'))
-    depends_on('py-numpy', type=('run'))
+    depends_on("py-antspyx@0.2.4", type=("run"))
```

Comment:
All dependencies here and below should actually be `"build", "run"`

Reply:
Is that something new? Weren't all normal python dependencies usually just listed with `type=("run")`?

Reply:
Didn't pay much attention to it before (and used both normally), but Gianluca pointed out at some point: https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#python-dependencies

> These packages are required for building and installation. You can add them with type=('build', 'run').

Kind of a safeguard against `pip` downloading anything required during installation with Spack.

Reply:
Fair enough, thanks for the reference.

Changed in 1c493704d7